src/components/TankSkeletons/TankSkeleton.module.scss

```diff
@@ -1,19 +1,19 @@
 .TankSkeleton {
+  --tank-skeleton-bg-x: calc(0px - var(--bg-double-block-size) * 40);
+  --tank-skeleton-bg-y: calc(0px - var(--bg-double-block-size) * 24);
   width: var(--block-size);
   height: var(--block-size);
+  background: black url("../../assets/images/sprite-big.png") no-repeat
+  var(--tank-skeleton-bg-x) var(--tank-skeleton-bg-y);
```

Comment:
- [ ] Add `/* background-position */` comment here to add a bit of clarity here.